packages/api/app/Exceptions/InvalidAccessException.js

```diff
@@ -0,0 +1,17 @@
+const { LogicalException } = require('@adonisjs/generic-exceptions');
+
+const message = 'The item is in an status where modifications are disallowed';
+const status = 401;
```

Comment:
Penso que o status code correto nesse caso seria o 403 (Forbidden). Sabemos quem é o usuário mas, por não possuir a permissão necessária, o acesso foi negado.

401 Unauthorized: referente a autenticação (apesar do nome)
403 Forbidden: referente a autorização

https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses

Reply:
it's confusing that the one called `Unauthorized` is actually for authentication and not authorization 🤦‍♂️ 

But yeah, typically `403` is meant to be used for authorization.